### ISSUE TITLE
Import from BitWarden fails

### ISSUE THREAD

**Issue by mihha** [buttercup/buttercup-desktop#820]
Hi,

today I installed the latest version and tried to import JSON file exported from BitWarden.

It fails with the following error

![image](https://user-images.githubusercontent.com/3791124/66732633-fb2dde80-ee5c-11e9-9e75-49a9468565fe.png)

I have the latest version of BitWarden and latest version of Buttercup

No idea how to proceed :-(

**Comment by perry-mitchell** [comment]
Hmm, I wonder if it could be a new format or something we've missed? Can you create a **sample vault** and export that to share here? We can do testing with that perhaps.

**Comment by perry-mitchell** [comment]
Yes it’d need to be a sample Bitwarden export that breaks so we can develop a fix - I don’t currently have bitwarden installed, so this would speed up the process.

Sounds strange that Bitwarden wouldn’t permit multiple vaults however..

**Comment by mihha** [comment]
OK

I managed to create a redacted version of my export from BitWarden so you can take a look
[bitwarden_export_20191016204943.zip](https://github.com/buttercup/buttercup-desktop/files/3735950/bitwarden_export_20191016204943.zip)
